html-formatter/javascript/check.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
+const path = process.argv[2]
+const html = fs.readFileSync(path, 'utf-8')
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.setContent(html)
+  await page.pdf({path: 'hn.pdf', format: 'A4'});
+  const dynamicHTML = await page.evaluate(() => {
+    const content = document.querySelector('.cucumber-react')
```

Comment:
Not sure if there could be errors in lower components (e.g. markdown) but the root one still render enough for this to pass?

To solve a similar problem before I've queried the console for errors - I could always do a follow up if you think it's a good idea.

Reply:
Yes, inspecting console errors could be useful too. This check does not guarantee absence of errors, but it does detect presence of errors. I think that's a good start.

But we have a problem with CircleCI launching puppeteer.....

Reply:
I've seen this before. They started requiring some libraries on the host system from 3.0.0 inc. `libgbm`.